### ISSUE TITLE
Notification payload changes

### ISSUE THREAD

**Issue by heisler98** [tiberloop/flaskchat#70]
Currently, notifications exist like this:
```
"New Message"
username: message
```
I propose some change. One option is as follows:

For a DM:
```
"Name"
Message
```
For a room notification:
```
"Room"
@name: message
```

**Comment by heisler98** [comment]
Also, there should be some way for you to attach a content payload to a notification. It is not viewable by the user, but it will communicate important information to my app. For example, you tell me the room ID of the notification, and when the user taps on the notification, I show them the notifying room.

**Comment by heisler98** [comment]
I implemented this change in PR #82 and will be testing shortly.
